client/src/modules/list/model/reducer.js

```diff
@@ -37,6 +34,39 @@ const items = (state, action) => {
             : item
         )
       };
+    case ItemActionTypes.ADD_DESCRIPTION_SUCCESS: {
+      const {
+        payload: { description, itemId }
+      } = action;
+
+      return {
+        ...state,
+        items: state.items.map(item =>
```

Comment:
What do you think about storing items collection in redux as an object which properties that are items' identifiers like in list and cohort case? We will have quick access to the specific item and we won't have to map overall collection?

Reply:
yes, we can...but I dont have time now to change this everywhere in reducers actions etc....
We can make separate task to refactore this